### DISCUSSION TITLE
hard to find `login` when there are many comments

### DISCUSSION THREAD

**Discussion by makerjackie** [giscus/giscus#489] [Q&A] [answered]
I wonder if there is a better hints to info user `you need to login first to comment` from the beginning, instead of scroll to the end of page to find `sign with github` button
![image](https://user-images.githubusercontent.com/18050469/162695746-1ea759a9-e03c-4b6b-aff5-6453f214c4c4.png)

**Comment by laymonage** [comment, marked as answer]
If you hover over the *button*, **not the number**, it'll show the message:

<img width="246" alt="image" src="https://user-images.githubusercontent.com/6379424/162703039-0437d089-df18-490b-9ca3-14fca946cbf1.png">

If this really bothers you, I recommend placing the comments box above the comments so that users immediately see the sign-in button before the comments.

<img width="711" alt="image" src="https://user-images.githubusercontent.com/6379424/162702942-81126e55-7bde-4902-9d6c-c122a5731f5b.png">

Result:

<img width="785" alt="image" src="https://user-images.githubusercontent.com/6379424/162703345-07b94f7d-8e43-498e-9529-c57f95a66be9.png">

**Reply by makerjackie** [reply]
thx for your reply, it would be great to show `you need to login` wherever you hover over
